### PR TITLE
add siddhant094 information

### PR DESCRIPTION
**Pull request by siddhant094**

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -149,7 +149,7 @@
       "batch": 2020,
       "picture_url": "https://i.ibb.co/4Rb9RRT/myimage.jpg",
       "twitter": "https://twitter.com/Akhiles63114004",
-      "github":"https://github.com/AkhileshManda",
```
Comment (nlok5923) [inline data.json:152]: @siddhant094 please revert all the unncessary changes.

**Review by nlok5923 — CHANGES_REQUESTED** [review]
PTAL

**Comment by nlok5923** [comment]
@siddhant094 still there are lot of unnecessary changes do remove it.

**Comment by siddhant094** [comment]
@nlok5923 Yes, i will open another pr by evening , by that time I am closing this pr

**Pull request closed without merging**